planckton/init.py

```diff
@@ -186,6 +187,40 @@ def pack(self, box_expand_factor=5):
             typed_system.strip([a.atomic_number == 1 for a in pmd_system.atoms])
         return typed_system
 
+    def stack(self, separation=0.7):
+        """Stack compounds in a single layer
```

Comment:
Add a period at the end of this sentence

Reply:
Yikes I didn't think about these things when writing the docstring. Fixed it, looks like pre-commit CI is happy.